### DISCUSSION TITLE
[Question] How to create persistent data in submods

### DISCUSSION THREAD

**Discussion by zwartkop2** [Monika-After-Story/MonikaModDev#8275] [QA - code and content creation (technical)]
I'm trying to make a submod for personal use. Is there any documentation on how to properly create player model persistent data?

**Comment by Booplicate** [comment]
I'd avoid using player model system for submods, it's possible to cause conflicts with the mod. But if you need to save something in your own persistent var:
```renpy
$ persistent._my_submod_var_name = True
```
